app/assets/javascripts/dictionary.js.coffee

```diff
@@ -30,17 +30,22 @@ $ ->
     size = $(window).width()
     divWidth = $('body > div.container').width()
     margin = (size - divWidth) / 2
-    $("#pdf-guide-sidenav").css("right",margin)
-    $("#pdf-guide-sidenav").css("top", "10px")
+    $("#pdf-guide-sidenav").css("right", margin)
+    $("#pdf-guide-sidenav").css("position", "")
   )
   sidenav.on('affix-top.bs.affix', ->
-      $("#pdf-guide-sidenav").css("right","0")
-      $("#pdf-guide-sidenav").css("top","0")
+    $("#pdf-guide-sidenav").css("right", "0")
+  )
+  sidenav.on('affix-bottom.bs.affix', ->
+    $("#pdf-guide-sidenav").css("right", "0")
   )
   # starts bootstrap affix
   sidenav.affix({
     offset: {
-      top: -> (this.top = $('#pdf-guide-sidenav').offset().top),
-      bottom: -> (this.bottom = $('img.footer-logo').outerHeight(true))
+      top: $('#pdf-guide-sidenav').offset().top,
```

Comment:
Yeah, I was thinking about adding such a line, because this script makes no sense on other pages.

I thought odd that this line would cause an error, I just cleaned it up a little. The former code was based on an example which was a bit overkill for this.